backend/src/app.ts

```diff
@@ -0,0 +1,8 @@
+import Duxcore from './Duxcore';
+import jwt from "jsonwebtoken";
+
+const app = new Duxcore();
+
+app.start().then(async() => {
+	console.log("Duxcore Backend Started.");
```

Comment:
Maybe the message should be something cooler like 
```console.log(`🎛️ Duxcore backend started at ${process.env.SESSION_REDIS_ADDRESS}.`);```

Reply:
I am not worried about the messages at the moment because I am just trying to get a working backend.  Once that is done I'll go in and add loggers and whatnot.

Reply:
oh yeah that's fair 👍🏼